src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -367,8 +361,6 @@ public unsafe int X
             get => Location.X;
             set
             {
-                _location.X = value;
```

Comment:
I do not get why this line was removed

Reply:
Because the location does get set by the OnMove event. If we'd want the changes to be immediate(which we perhaps do), then the tests which test for location change and stuff like that are unecessary, as they do test if setting of a field did work.